lib/stub-generator.js

```diff
@@ -101,14 +97,35 @@ StubGenerator.prototype.build = function() {
   debug('build in %dms', Date.now() - start);
 };
 
+// TODO: Extract
 function rewriteImports(content, data) {
-  // TODO: real AST printing might be a good plan..
+  var lines = content.split('\n');
+  var offsets = [];
+
   Object.keys(data).forEach(function(name) {
+    var line = data[name].start.line - 1;
+
+    if (offsets[line] === undefined) {
+      offsets[line] = 0;
+    }
+
+    var start = data[name].start.column + offsets[line];
+    var end = data[name].end.column + offsets[line];
     var version = data[name].version;
-    content = content.replace(new RegExp('["]npm:' + name + '["]', 'g'), '"npm:' + name + '@' + version + '"');
-    content = content.replace(new RegExp('[\']npm:' + name + '[\']', 'g'), '\'npm:' + name + '@' + version + '\'');
+
+    var pre = lines[line].substring(0, start);
+    var moduleString = '"npm:' + name + '@' + version + '"';
+    var post = lines[line].substring(end);
```

Comment:
may want to consider using: https://github.com/Rich-Harris/magic-string if we are worried re: losing source-maps